### PR TITLE
 Adding notes and organNames

### PR DESCRIPTION
**Pull request by vutukuriajay2241**

### CODE REVIEW

File: projects/v2/src/app/actions/tree.actions.ts [hunk 1]
```diff
@@ -52,5 +52,5 @@ export class UpdateBottomSheetData {
 
 export class UpdateLinksData {
   static readonly type = '[UPDATE] Links data';
-  constructor(public AS_CT: number, public CT_B: number, public AS_AS?: number) {}
+  constructor(public AS_CT: number, public CT_B: number,  public AS_CT_organWise: object, public CT_B_organWise: object, public AS_AS?: number,public AS_AS_organWise?: object) {}
```
Comment (bherr2) [inline projects/v2/src/app/actions/tree.actions.ts:55]: 2 spaces => 1 space

File: projects/v2/src/app/store/tree.state.ts [hunk 2]
```diff
@@ -66,6 +66,9 @@ export class TreeStateModel {
     AS_CT: number;
     CT_B: number;
     AS_AS: number;
+    AS_CT_organWise: object;
```
Comment (bherr2) [inline projects/v2/src/app/store/tree.state.ts:69]: Tighten up types here. Should be `Record<string, number>` right?

**Review by bherr2 — CHANGES_REQUESTED** [review]
A couple minor things, but otherwise fine

File: projects/v2/src/app/actions/tree.actions.ts (hunk 1 above)
Comment (vutukuriajay2241) [reply projects/v2/src/app/actions/tree.actions.ts:55]: I didnt get this??

**Review by bherr2 — APPROVED** [review]

**Pull request merged**